src/Guzzle6HttpAdapter.php

```diff
@@ -45,24 +45,40 @@ public function sendRequest(RequestInterface $request)
     {
         try {
             return $this->client->send($request);
-        } catch (RequestException $e) {
-            throw $this->createException($e);
+        } catch (Guzzle\SeekException $e) {
+            throw new RequestException($e->getMessage(), $request, $e);
+        } catch (Guzzle\TransferException $e) {
```

Comment:
In case they introduce new ones. Also, I am not sure that plugins don't introduce new ones.